Frends.FTP.DownloadFiles/Frends.FTP.DownloadFiles/Definitions/RenamingPolicy.cs

```diff
@@ -3,281 +3,264 @@
 using System.IO;
```

Comment:
https://github.com/FrendsPlatform/Frends.FTP/blob/3b2b2fbd96e793fb58e0ddc3e006dc3f6fcb985c/Frends.FTP.DownloadFiles/Frends.FTP.DownloadFiles/Definitions/RenamingPolicy.cs#L1-L3
You are not using these namespaces, so they can be removed.